app/controllers/v1/sessions_controller.rb

```diff
@@ -1,5 +1,5 @@
 class V1::SessionsController < ApplicationController
-  before_action :require_login, except: :create
+  before_action :require_login, except: %i(create is_logged_in?)
```

Comment:
You don't need to add `is_logged_in?` to the exceptions, as we actually want sorcery to raise not authorized error if the user is not logged is, as it's handed in ApplicationController#not_authenticated.
That's enough for the frontend.
So, when user is not authenticated, on the frontend we get error with status 401, and if it is logged in, we get serialized current user.